### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by TaprootFreak**

### CODE REVIEW

File: components/PageEcoSystem/DistributionZCHF.tsx
Comment (davidleomay) [inline components/PageEcoSystem/DistributionZCHF.tsx:1]: Rename?

File: components/PageEquity/EquityInteractionWithFPSWFPS.tsx
Comment (davidleomay) [inline components/PageEquity/EquityInteractionWithFPSWFPS.tsx:1]: Rename?

File: components/PageEquity/EquityInteractionWithWFPSRedeem.tsx
Comment (davidleomay) [inline components/PageEquity/EquityInteractionWithWFPSRedeem.tsx:1]: Rename?

File: components/PageEquity/EquityInteractionWithZCHFFPS.tsx
Comment (davidleomay) [inline components/PageEquity/EquityInteractionWithZCHFFPS.tsx:1]: Rename?

**Comment by TaprootFreak** [comment]
we take the renaming into a separate PR

**Review by Danswar — APPROVED** [review]

**Review by davidleomay — APPROVED** [review]

**Pull request merged**